metasrv/tests/it/grpc/metasrv_grpc_api.rs

```diff
@@ -412,6 +397,17 @@ async fn test_auto_sync_addr() -> anyhow::Result<()> {
 
         start_metasrv_with_context(&mut tc3).await?;
 
+        let g = tc3.grpc_srv.as_ref().unwrap();
+        let meta_node = g.get_meta_node();
+
+        let metrics = meta_node
+            .raft
+            .wait(Some(Duration::from_millis(30_000)))
```

Comment:
```suggestion
            .wait(Some(Duration::from_secs(30)))
```

Reply:
BTW, it's quite annoying to wait/sleep in testing code. Is it possible for `raft` to implement something like `notify` or `watch` (test only)?